src/getCypressElementCoordinates.ts

```diff
@@ -73,7 +76,10 @@ export function getCypressElementCoordinates(
   // This breaks the coordinates system of the whole tab, so we need to compensate the scale value.
   const appFrameScale = appWidth / cypressAppFrame.offsetWidth;
 
-  htmlElement.scrollIntoView({ block: "center" });
+  if (scrollBehavior) {
```

Comment:
Honestly it's not how cypress works in reality. We should not run scrolling at all if element is displayed on the screen.

Reply:
We could throw this out and just check if the element is already on screen before calling scrollIntoView?

Reply:
No it makes sense in terms of cypress compatibility. But cypress driver is not scrolling if the element is on the screen - we should be same and I think that was a root of issue?

Reply:
ah I see what you're saying- yeah, that makes sense. I'll push up an update to cover that

- [ ] Don't scroll at all if it's not needed - mimic cypress

Reply:
I looked into this a bit more, and I don't think changing it to not scroll if it's already on the screen would match Cypress's behavior.

Cypress does actually always scroll the element by default, even if it's already in the viewport. It does so to make sure that elements are always in the same positions when the tests run.

the `cy.click` command calls `$actionability.verify` [here](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/actions/click.js#L176). The other actions do the same.

And `$actionability.verify` calls `scrollIntoView` [unless you pass `scrollBehavior: false`](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/actionability.js#L328-L338)
 
So the Cypress driver _does_ scroll the element if it's already on the screen- the issue here is just that we can't control the scrollBehavior when using `realHover` or `realClick`